CONTRIBUTING.md

```diff
@@ -0,0 +1,125 @@
+# Contributing
+
+## General Workflow
+
+1. Fork the repo
+1. Cut a namespaced feature branch from master
+1. Make commits to your feature branch. Prefix each commit like so:
+1. When you've finished with your fix or feature, merge upstream changes into your branch. submit a pull request directly to master. Include a description of your changes.
```

Comment:
We can do that. I can see it being a good way to track conversations of a specific feature being implemented.